documentation/reference/revision-resource-compatibility.md

```diff
@@ -0,0 +1,21 @@
+# Compatible Charm Revisions and Resources
+
+Deploying charms from the `latest/stable` channel *should* guarantee 
+compatibility between charm revision and resource.
+
+However, if you are deploying earlier revisions of the charm please
```

Comment:
"Revisions" or "versions"? I am not sure about the technical term in this case

Reply:
`revisions` = the revision of the charm code, `version` = the applications version. So I think revision is the correct use here